### PR TITLE
Consume bootstrap config generated by OSM

### PR DESCRIPTION
**Pull request by ahmedwaleedmalik**
**What this PR does / why we need it**:
This is a supporting PR for https://github.com/kubermatic/operating-system-manager/issues/95, which intends to move the bootstrapping script from machine controller to OSM.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Optional Release Note**:

```release-note
Bootstrap config has been moved to OSM
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -36,20 +36,20 @@ require (
...
 	gomodules.xyz/jsonpatch/v2 v2.2.0
 	google.golang.org/api v0.74.0
 	google.golang.org/grpc v1.45.0
 	gopkg.in/gcfg.v1 v1.2.3
 	gopkg.in/yaml.v3 v3.0.1
-	k8c.io/operating-system-manager v0.4.4
-	k8s.io/api v0.24.0
-	k8s.io/apiextensions-apiserver v0.24.0
-	k8s.io/apimachinery v0.24.0
+	k8c.io/operating-system-manager v0.4.5-0.20220628074909-3d9712f46bae
```
Comment (ahmedwaleedmalik) [inline go.mod:46]: This is intentionally pinned to a commit hash. I need to test this before the new OSM and machine-controller release. Once tested we'll pin this to a server tag.

**Review by moadqassem — APPROVED** [review]
/approve
/lgtm

**Pull request merged**